services/ark-dashboard/ark-dashboard/app/(dashboard)/query/[id]/page.tsx

```diff
@@ -246,77 +266,58 @@ function QueryDetailContent() {
 
   // Extract example from JSON schema
   const getSchemaExample = (schema: Record<string, unknown>): string | null => {
-    // Look for explicit examples
     if (schema.example) {
-      return typeof schema.example === 'string' 
-        ? schema.example 
+      return typeof schema.example === 'string'
+        ? schema.example
         : JSON.stringify(schema.example, null, 2)
     }
 
-    // Look for examples in properties or generate empty structure
     if (schema.type === 'object' && schema.properties) {
       const properties = schema.properties as Record<string, Record<string, unknown>>
       const example: Record<string, unknown> = {}
 
       for (const [key, prop] of Object.entries(properties)) {
-        if (prop.example !== undefined) {
-          example[key] = prop.example
-        } else if (prop.default !== undefined) {
-          example[key] = prop.default
-        } else {
-          // Generate empty placeholder based on type
-          if (prop.type === 'string') {
-            example[key] = ""
-          } else if (prop.type === 'number' || prop.type === 'integer') {
-            example[key] = 0
-          } else if (prop.type === 'boolean') {
-            example[key] = false
-          } else if (prop.type === 'array') {
-            example[key] = []
-          } else if (prop.type === 'object') {
-            example[key] = {}
-          } else {
-            example[key] = null
-          }
+        if (prop.example !== undefined) example[key] = prop.example
```

Comment:
Can we discuss this i think we need to find a clearer way ot extract into some kind of reusable helper it is not clear what this is doing